src/lib/hooks/swap/useSwapInfo.tsx

```diff
@@ -152,6 +157,10 @@ const swapInfoAtom = atom<SwapInfo>({
   currencyAmounts: {},
   trade: { state: TradeState.INVALID },
   allowedSlippage: new Percent(0),
+  fee: {
+    fee: new Percent(0),
```

Comment:
This should be the default value for the atom, so import it from the file defining the atom.